.github/workflows/engine-and-cli-publish.yml

```diff
@@ -114,15 +114,15 @@ jobs:
             --aws-secret-access-key=env:AWS_SECRET_ACCESS_KEY \
             --aws-region=env:AWS_REGION \
             --aws-bucket=env:AWS_BUCKET \
-            --artefacts-fqdn=env:ARTEFACTS_FQDN
+            --artefacts-fqdn="$ARTEFACTS_FQDN"
```

Comment:
Don't we encourage the `env:ARTEFACTS_FQDN` format?

Reply:
I've updated the `artefactsFQDN` in the CLI to be a string arg, not a secret anymore. This means that the `env:` form isn't supported there.